tasmoadmin/config.json

```diff
@@ -16,20 +16,23 @@
   "boot": "auto",
   "hassio_api": true,
   "hassio_role": "default",
+  "ingress": true,
+  "ingress_port": 8099,
```

Comment:
Is there a reason why the port is hardcoded for this add-on?

Reply:
The reasoning behind it is that the port is hard coded in the nginx ingress, so should hass.io ever change it's default port, it wouldn't matter as we have explicitly defined it here.